.gitignore

```diff
@@ -23,6 +23,8 @@ vendor/assets/bower_components/
 /db/schema.rb
 /public/system/*
 /coverage/
+/features/req_cache/*
```

Comment:
Oh, ok, thank you :) . I'll change this; when we were working on this PP that was only one idea which we came up with to avoid submitting all those files to remote again!